src/main/java/org/osiam/configuration/JacksonConfig.java

```diff
@@ -0,0 +1,20 @@
+package org.osiam.configuration;
+
+import com.fasterxml.jackson.core.Version;
+import com.fasterxml.jackson.databind.Module;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import org.osiam.resources.helper.UserDeserializer;
+import org.osiam.resources.scim.User;
+import org.springframework.boot.autoconfigure.web.WebMvcAutoConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class JacksonConfig extends WebMvcAutoConfiguration {
```

Comment:
Don't inherit from `WebMvcAutoConfiguration`. I don't think it's a good idea. Why is it necessary anyway? The docs state:

> Another way to customize Jackson is to add beans of type com.fasterxml.jackson.databind.Module to your context. They will be registered with every bean of type ObjectMapper, providing a global mechanism for contributing custom modules when you add new features to your application.

(http://docs.spring.io/spring-boot/docs/1.3.2.RELEASE/reference/htmlsingle/#howto-customize-the-jackson-objectmapper)

So it should be enough to add the `@Bean` declaration to `OAuth2ResourceServerConfig` or some other place (or just leave it in this `@Configuration` class).